src/pages/event.js

```diff
@@ -127,6 +143,24 @@ export function Event() {
     }
   }, [ensNames]) /* eslint-disable-line react-hooks/exhaustive-deps */
 
+  useEffect(() => {
+    if (loadingEvent === 'succeeded' && csvDownloadIsOnLastStep()) {
```

Comment:
We need this to be a constant `'succeeded'`. Move all values of loadingEvent to constants

Reply:
we can use arrow functions to handle these states
something like that

 ```js
 const isLoadingEvent = () => loadingEvent === LOADING_SUCCEEDED;
```

Reply:
https://github.com/poap-xyz/poap-gallery/pull/27/commits/7e947ea870f86e9a75318d72ab194ded2da569a8